packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -65,16 +65,7 @@
         <tr v-if="showSharedBy" data-testid="shared-by">
           <th scope="col" class="oc-pr-s oc-font-semibold" v-text="$gettext('Shared by')" />
           <td>
-            <span v-text="sharedByDisplayName" />
-          </td>
-        </tr>
-        <tr v-if="ownerDisplayName" data-testid="ownerDisplayName">
```

Comment:
Do we still need to show the owner for CERN? 🤔

Reply:
> Do we still need to show the owner for CERN?

Probably depends on how they will be using sharing NG 😅 

> Maybe we can keep the owner, but hide it if it's identical with the share owner.

In which scenario would they be different though? Because for project spaces we don't have file owners. And in the personal space we don't show the file owner anyway because it's always the current user.

Reply:
You might have an owner without any share. CERN allows access to files of other users without a share (shared root folder without an ownCloud share).